src/test/java/com/google/jenkins/plugins/storage/integration/DownloadStepPipelineIT.java

```diff
@@ -79,6 +81,9 @@ public void testDownloadStepSuccessful() throws Exception {
     assertNotNull(run);
     jenkinsRule.assertBuildStatus(Result.SUCCESS, jenkinsRule.waitForCompletion(run));
     dumpLog(LOGGER, run);
+
+    VirtualFile archivedFile = run.getArtifactManager().root().child(pattern);
+    assertTrue(archivedFile.exists());
```

Comment:
This particular assert is similar to the ones used in other projects:
- https://github.com/jenkinsci/workflow-api-plugin/blob/de2644ed28d708a76ea116b8172fc3eb289796dd/src/test/java/org/jenkinsci/plugins/workflow/ArtifactManagerTest.java#L147-L151